yt_dlp/extractor/pokergo.py

```diff
@@ -65,7 +66,11 @@ def _real_extract(self, url):
             'width': image.get('width'),
             'height': image.get('height'),
         } for image in data_json.get('images') or [] if image.get('url')]
-        series_json = next(dct for dct in data_json.get('show_tags') or [] if dct.get('video_id') == video_id) or {}
+
+        series_json = {}
+        with contextlib.suppress(StopIteration):
+            # sometimes API returns no show_tags for the video
+            series_json = next(dct for dct in data_json.get('show_tags') or [] if dct.get('video_id') == id) or {}
```

Comment:
I don't have an account, so I can't test this, but how about:
```suggestion
        series_json = traverse_obj(data_json, ('show_tags', lambda _, v: v['video_id'] == video_id, any)) or {}
```
Requires importing `from ..utils.traversal import traverse_obj`

Reply:
@seproDev Thanks for the suggestion, I've updated PR